static/app/components/profiling/flamegraph/flamegraphZoomView.tsx

```diff
@@ -858,19 +861,19 @@ function handleFlamegraphKeyboardNavigation(
 
   const key = evt.shiftKey ? `Shift+${evt.key}` : evt.key;
   let direction = keyDirectionMap[key];
+
+  // if the key is not mapped, don't handle it
   if (!direction) {
-    return currentFrame;
```

Comment:
This fixes the issue, but it does not fix the problem in computeConfigViewWithStrategy, that is done below